storagenode/storagenodedb/database.go

```diff
@@ -232,22 +205,35 @@ func (db *DB) Close() error {
 		db.ndb.Close(),
 		db.adb.Close(),
 
-		db.versionsDB.Close(),
-		db.v0PieceInfoDB.Close(),
-		db.bandwidthDB.Close(),
-		db.ordersDB.Close(),
-		db.pieceExpirationDB.Close(),
-		db.pieceSpaceUsedDB.Close(),
-		db.reputationDB.Close(),
-		db.storageUsageDB.Close(),
-		db.usedSerialsDB.Close(),
-		db.satellitesDB.Close(),
+		db.closeDatabases(),
 	)
 }
 
-// Versions returns the instance of the versions database.
-func (db *DB) Versions() SQLDB {
-	return db.versionsDB
+// closeDatabases closes all the SQLite database connections and removes them from the associated maps.
+func (db *DB) closeDatabases() error {
+	var err error
+
+	for k := range db.sqlDatabases {
+		_ = errs.Combine(err, db.closeDatabase(k))
```

Comment:
don't we need `err =` here? I believe this will discard all the errors ... ?

Reply:
Fixed!